sanowret-crystal.lic

```diff
@@ -0,0 +1,73 @@
+=begin
+	Runs in the background, gazing or exhaling your sanowret crystal. Checks for concentration and current mindstate.
+
+	author: NBSL
+	game: dr
+  version: 0.1 (2017-06-12)
+
+  changelog:
+    0.1 (2017-06-12)
+     * Initial Release
+=end
+
+custom_require.call(%w[common drinfomon])
+
+
+class SanowretCrystal
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'run', regex: /run/i, description: 'Single run of the script', optional: true }
+      ]
+    ]
+    @invalid_room = nil
+    args = parse_args(arg_definitions)
+    check_crystal if args.run
+    passive if args.run.nil?
+  end
+
+  def use_crystal
+    return if @invalid_room == Room.current.id
+    if DRSkill.getxp('Arcana') <= 10
+      reponse = DRC.bput("gaze sanowret crystal", "A soft light blossoms in the very center of the crystal, and begins to fill your mind with the knowledge of \w+.", "However, nothing much else happens, as you lack the concentration to focus.", "This is not a good place for that.")
```

Comment:
The only issue I've seen is typeahead. I don't know of a way to handle that. 

Other than that, gaze works during RT. Exhale forces you to wait.